ledger/ledger-api-health/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2019 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_library",
+    "da_scala_test_suite",
+)
+
+da_scala_library(
```

Comment:
I didn't check it yet, writing here before I forget: did you add the required lines on `//release/artifacts.yaml`?

Reply:
Yes, you did. :slightly_smiling_face: